### PR TITLE
Updating Spark version in test application specs

### PR DESCRIPTION
**Pull request by akirillov**
### What changes were proposed in this pull request?
This PR updates Spark versions used in tests to match the latest released version.

### Why are the changes needed?
To avoid usage of different Spark versions in tests,  scale tests, and user-facing examples.

### How were the changes tested?
Integration tests from this repo

### CODE REVIEW

File: scale-tests/templates/gensort-application.tmpl [hunk 1]
```diff
@@ -5,10 +5,10 @@ metadata:
 spec:
   type: Scala
   mode: cluster
-  image: mesosphere/spark:2.4.3-bin-hadoop2.9-k8s
+  image: mesosphere/spark:2.4.5-bin-hadoop2.9-k8s
```
Comment (alembiewski) [inline scale-tests/templates/gensort-application.tmpl]:
```suggestion
  image: mesosphere/spark:spark-2.4.5-bin-hadoop2.9-k8s
```

File: scale-tests/templates/scale-test-application.tmpl [hunk 2]
```diff
@@ -5,32 +5,32 @@ metadata:
 spec:
   type: Scala
   mode: cluster
-  image: mesosphere/spark:2.4.3-bin-hadoop2.9-k8s
+  image: mesosphere/spark:2.4.5-bin-hadoop2.9-k8s
```
Comment (alembiewski) [inline scale-tests/templates/scale-test-application.tmpl]:
```suggestion
  image: mesosphere/spark:spark-2.4.5-bin-hadoop2.9-k8s
```

File: scale-tests/templates/sort-application.tmpl [hunk 3]
```diff
@@ -5,10 +5,10 @@ metadata:
 spec:
   type: Scala
   mode: cluster
-  image: mesosphere/spark:2.4.3-bin-hadoop2.9-k8s
+  image: mesosphere/spark:2.4.5-bin-hadoop2.9-k8s
```
Comment (alembiewski) [inline scale-tests/templates/sort-application.tmpl]:
```suggestion
  image: mesosphere/spark:spark-2.4.5-bin-hadoop2.9-k8s
```

**Comment by akirillov** [comment]
thanks for catching this, @alembiewski!

**Pull request merged**